### ISSUE TITLE
Prefix all available commands with GitSavvy for discoverability

### ISSUE THREAD

**Issue by gerardroche** [timbrel/GitSavvy#1756]
Some commands are prefixed "github:" e.g. `github: open file on remote`. I want to suggest prefixing all available commands with **GitSavvy: * for discoverability. 

[braver](https://github.com/wbond/package_control_channel/pull/8756#issuecomment-1557701686) just made me aware of the command `github: open file on remote`. I had no idea it existed. I would have know if the command was prefixed "GitSavvy" because to see what commands were available that's what I checked.

**Comment by gerardroche** [comment]
I wouldn't have thought it would be very much disruptive because of fuzzy search in the command palette. Users typing `github: ...` should generally still get the same command. I was thinking something more like `gitsavvy: github: open file/issue` on remote` to avoid that exact issue. Ideally it would probably be more like `gitsavvy: open github file/issue on remote`, but I'd go with the former to avoid the disruption issue i.e. users type `gh: ...` should still get the same command as before.

**Comment by kaste** [comment]
Okay, I generally dislike the double prefix `gitsavvy: github: `.  Then: `gitsavvy: open github file on remote`  probably doesn't make much sense.  I never understood "Open file on remote" anyway (but learned to deal with it) as it is "open file in browser" actually.  I *do* like `open file on github` of course because it is the better description in all regards.  

But although I like the natural language style, "abort merge", "continue rebase", "open file on github", we typically use the reversed style in GitSavvy.  (We do `github: create pull request`, you suggest `Create pull request on GitHub`.)  This reversed style leads to some grouping like 

![image](https://github.com/timbrel/GitSavvy/assets/8558/f75ad3b4-95bf-42d9-b888-659c558ccbf9)

So a "pull with rebase" is clearly not a "rebase" action.    

The prefix `GitSavvy` is currently reserved for internal commands.  Typically users don't need them (except the Preferences).

![image](https://github.com/timbrel/GitSavvy/assets/8558/2017b263-5794-4f17-bd9c-cf8b7cae8b89)

Then we "register" also `git: ` and the hub stuff.  E.g. for GitHub:

![image](https://github.com/timbrel/GitSavvy/assets/8558/158d9891-59b5-488e-bafa-faafa2be0d71)

I would like to get more input because in general I *really* like the prefix style esp. for discoverability.  What can I do with "github", just type `github: `.  (Of course we don't mention this style in the README 🙄.)  Also in this list only "open file on remote" is completely misleading as "remote" typically refers to a local configuration thing.  For example: "add fork as remote" is somewhat correct as the users chooses one of the forks hosted on github and adds it to the local configuration.  "open file in the browser", "open file on the web". 

![image](https://github.com/timbrel/GitSavvy/assets/8558/3b223fd3-9ec0-4983-adec-c88722b819cb)

**Comment by gerardroche** [comment]
Yes. Now I know there are also some commands prefixed "git: ". I like to be able to see what package a command belongs to, which is why I like packages to prefix all their commands with the name of their package. 

Perhaps there is a Sublime Text Command Palette feature in there somewhere e.g. the Command Palette prefixes all the commands with the package name, something like how it suffixes the key binding. For backwards compatibility it could strip out any existing prefix so it's not duplicated.

The "github: " prefix seems a bit clunky to me. All of those commands would apply to gitlab, bitbucket, etc. and if that's not a feature I have to assume it will be so "git: " makes sense me even if those services are not supported yet.

All the commands are lowercase too, which seems at odds to the general format of the Command Palette. Perhaps Sublime Text should provide a best practice guide for naming conventions to encourage consistency.
